src/LoginForm.jsx

```diff
@@ -1,14 +1,18 @@
-export default function LoginForm({ onSubmit }) {
+export default function LoginForm({ onChange, onSubmit }) {
+  function handleChange(event) {
+    const { target: { name, value } } = event;
+    onChange({ name, value });
+  }
   return (
     <>
       <div>
         <label htmlFor="login-email">E-mail</label>
-        <input type="email" id="login-email" />
+        <input name="email" onChange={handleChange} type="email" id="login-email" />
```

Comment:
```suggestion
        <input
          type="email"
          id="login-email"
          name="email"
          onChange={handleChange}
        />
```

이렇게 표현하면 element 파악이 더 쉬울 것 같아요

Reply:
### 수정했습니다!
줄바꿈을 해주었습니다!!